cmd/snap/cmd_auto_import.go

```diff
@@ -153,6 +148,20 @@ func init() {
 	cmd.hidden = true
 }
 
+func isOwned() (bool, error) {
```

Comment:
Owned => Managed.. I _think_ that's the final terminology we agreed to, although we've been speaking about "owned" frequently.